core/api/type/type_proto/keymaster.proto

```diff
@@ -48,8 +49,9 @@ message SigningKey {
 message VerifyingKey {
   // KeyStatus defines a key status.
   enum KeyStatus {
-    ACTIVE = 0;      // indicates a key that is currently being used.
-    DEPRECATED = 1;  // indicates a key that is not in use.
+    UKNOWN = 0;
+    ACTIVE = 1;      // indicates a key that is currently being used.
+    DEPRECATED = 3;  // indicates a key that is not in use.
```

Comment:
I'm not sure of the benefit of aligning the enum values like this.  I'd just use 2 here.  There should not be any casting between the two KeyStatus types in any case!

Reply:
Yeah, no casting indeed.